utils/spacewalk-common-channels.ini

```diff
@@ -3670,6 +3670,179 @@ checksum = sha256
 base_channels = ubuntu-22.04-pool-amd64-uyuni
 repo_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Master:/Ubuntu2204-Uyuni-Client-Tools/xUbuntu_22.04/
 
+[ubuntu-2404-pool-amd64-uyuni]
+label     = ubuntu-24.04-pool-amd64-uyuni
```

Comment:
AFAIK we got some reports in the past that we should stick to one type of labeling. Here we use a "." for the version but nobody else we do it. So can we just remove the dot here as well?

@lucidd we need to check what we do in sumatoolbox.

Reply:
This dot notation is used for all Ubuntu versions in this file:
```
label    = ubuntu-16.04-pool-amd64
label    = ubuntu-16.04-pool-amd64-uyuni
label    = ubuntu-18.04-pool-amd64
label    = ubuntu-18.04-pool-amd64-uyuni
label    = ubuntu-20.04-pool-amd64-uyuni
label    = ubuntu-22.04-pool-amd64-uyuni
label    = ubuntu-24.04-pool-amd64-uyuni
```

Reply:
Yes, we did that in the past, but this does not mean that it is good.
It seems that this pool channel is the only channel label which uses "dots".
All others have it removed.
Why not starting with 24.04 having the pool channel also without dots ?

Reply:
@mcalmer we use ubuntu-2404 and have since 20.04 but i also noticed we got an inconsistency ourselves as client tools channels still use dot. we might want to fix that with 24.04